ads/how-to-guides/dynamic-backdrops.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 8
+---
+
+# Dynamic backdrops
+
+Some layouts, such as L-shape and Double Box, make use of a backdrop. These can either be specified on both the monetised stream and the ad break in 2 ways:
+- `backdropURI`: Optional URI containing the default back-drop to be used during the Double Box or L-shape ads.
```

Comment:
nitpick: @tjtheo I'd remove the hyphen in backdrop here and in the sentence below too.

I also would remove "either" in the sentence above.